scripts/us_eia/eia_860/power_plant.py

```diff
@@ -0,0 +1,241 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Processor for Schedule 2 of EIA Form 860 which includes Power Plant data.
+
```

Comment:
nit: should we update README https://github.com/datacommonsorg/data/tree/master/scripts/us_eia/eia_860 ?

Reply:
done